platform/node/README.md

```diff
@@ -9,7 +9,7 @@ Requires a modern C++ runtime that supports C++14.
 By default, installs binaries. On these platforms no additional dependencies are needed.
 
 - 64 bit macOS or 64 bit Linux
-- Node.js v4.x _(note: v5+ is known to have issues)_
+- Node.js v10.x
```

Comment:
We are only supporting + building with node v10 currently, so this drifted out of date